test/__init__.py

```diff
@@ -131,12 +131,12 @@ def decorate(fn):
         fn_name = fn.__name__
 
         def maybe(*args, **kw):
-            if predicate():
-                msg = "'%s' skipped: %s" % (fn_name, reason)
-                raise unittest.SkipTest(msg)
-            else:
+            if not predicate():
                 return fn(*args, **kw)
 
+            msg = "'%s' skipped: %s" % (fn_name, reason)
+            raise unittest.SkipTest(msg)
+
```

Comment:
**Michael Bourke** (bourke) wrote:

Can you help me understand this change? To my eyes, the original expresses the intent more clearly.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/mako/+/3319